omod/src/main/resources/config.xml

```diff
@@ -20,6 +20,12 @@
         <defaultValue>/spa</defaultValue>
         <description>The base URL from where Single Page Applications are served</description>
     </globalProperty>
+
+    <globalProperty>
+        <property>spa.frontend.directory</property>
+        <defaultValue>frontend</defaultValue>
+        <description>The directory that holds static files for the single-spa</description>
```

Comment:
I would drop 'single-' because it is somehow implied in spa. So i would say, 'The directory that holds static files for the single page applications'

Reply:
It's called [single-spa](https://single-spa.js.org/)

Reply:
@dkayiwa what is your response to brandones reply?

Reply:
Since the two of you feel like it should be single-spa, let us go with that. 😊

Reply:
Thanks